src/main/java/com/github/minecraftschurlimods/arsmagicalegacy/common/spell/PrefabSpellManager.java

```diff
@@ -80,5 +79,10 @@ public ItemStack makeSpell() {
             SpellItem.setSpellIcon(stack, icon());
             return stack;
         }
+
+        @Override
+        public int compareTo(IPrefabSpell o) {
```

Comment:
Why does it need to be comparable? Why by name and not by id?

Reply:
Because there is no id available. PrefabSpell only has the name, the spell and the icon resource location

Reply:
I sent you a patch to use the id instead of the name